src/components/TextInputWithLabel.js

```diff
@@ -53,16 +66,8 @@ const TextInputWithLabel = props => (
                 </TextLink>
             )}
         </View>
-        <TextInput
-            style={[
-                styles.textInput,
-                styles.mb1,
-                props.disabled ? styles.inputDisabled : undefined,
-                props.errorText ? styles.errorOutline : undefined,
-            ]}
-            // eslint-disable-next-line react/jsx-props-no-spreading
-            {..._.omit(props, ['label', 'errorText'])}
-        />
+        {/* eslint-disable-next-line react/jsx-props-no-spreading */}
+        <TextInputWithPrefix {...props} />
```

Comment:
I feel like we have this rule in place for exactly this kind of situation, right? Essentially, we want to explicitly define the props we're sending down to `TextInputWithPrefix` explicitly so we don't unintentionally send extra props

https://github.com/yannickcr/eslint-plugin-react/blob/master/docs/rules/jsx-props-no-spreading.md

Reply:
We're still just passing down most of the props, so I've updated it to pass props with any unnecessary ones omitted.

Quick question, should propTypes be defined at the highest level or at the lowest level they are needed? For instance, we pass a `disabled` prop to `TextInputWithLabel` which is passed directly down to `TextInputWithPrefix` where it is actually accessed. Should `disabled` go in the propTypes for `TextInputWithLabel` or `TextInputWithPrefix`? I'm thinking it should go with `TextInputWithLabel` since `TextInputWithPrefix` is only used by `TextInputWithLabel` internally.